client/src/bundles/UserForms.js

```diff
@@ -680,13 +684,14 @@ jQuery(document).ready(($) => {
     }
 
     // Display all the things that are hidden when JavaScript is disabled.
-    $('.userform-progress, .step-navigation').attr('aria-hidden', false).show();
+    $userform.find('.userform-progress, .step-navigation').attr('aria-hidden', false).show();
 
     // Extend classes with common functionality.
     $.extend(FormStep.prototype, commonMixin);
     $.extend(ErrorContainer.prototype, commonMixin);
 
-    userform = new UserForm($userform);
+    const userform = new UserForm($userform);
+    $userform.data('inst', userform);
```

Comment:
Short of refactoring this whole class to use DI for `ProgressBar` and `FormActions`, I can’t think of another way of doing this 😁